examples/src/main/scala/mhtml/examples/HelloWorld.scala

```diff
@@ -1,4 +1,5 @@
 package mhtml.examples
+import mhtml._
```

Comment:
My idea was to tailor the error message to a less experienced audience, for example frontend developers who are taking their first steps in Scala. That's why I thought the "troubleshooting" part might be smart and to explicitly mention that both Var and Rx are OK.

Veteran Scala developers should have an easier time figuring out what's happening without help from the error message.

I agree that Char and Boolean should be included, to support all literal types.
I also agree that we should say Seq[xml.Node] instead of Iterable[xml.Node]